packages/runtime-plugin-import-meta/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import { ModuleSystem } from './../../runtime/src/module-system';
+import type { FarmRuntimePlugin } from '@farmfe/runtime/src/plugin';
+
+export default <FarmRuntimePlugin>{
+  name: 'farm-runtime-import-meta',
+  _moduleSystem: {} as ModuleSystem,
+  bootstrap(system: ModuleSystem) {
+    this._moduleSystem = system;
+  },
+  moduleCreated(module) {
+    module.meta.env = {
+      ...process.env,
```

Comment:
Does this process.env contain all the environment variables of node? if so, we should not use full process.env.

Reply:
So should we still limit the variables in import.meta.env? At first, I didn't want to inherit process.env, but if developers customize some env, import.meta.env won't be available.